marimbabot_behavior/src/marimbabot_behavior/behavior_node.py

```diff
@@ -151,69 +206,71 @@ def callback_command(self, command_msg):
                 rospy.logwarn('No notes recognized.')
                 self.response_pub.publish('No notes recognized.')
 
-        # play the notes on the marimba using the UR5
-        elif command == 'marimbabot start playing':
+        # play the notes on the marimba using the UR5 or generate an audio preview
+        elif command == 'marimbabot start (playing|preview)':
```

Comment:
I don't think this works

Reply:
As this is a normal equal operation and not a regex

Reply:
You are right! Must have missed this. I fixed it now